HW6/repositories/order.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { OrderModel } from "../models/order.model.ts";
+import { CartEntity, DeliveryEntity, PaymentEntity } from "../types/index.ts";
+
+export const createNewOrder = (
+  cart: CartEntity,
+  payment?: PaymentEntity,
+  delivery?: DeliveryEntity,
+  comments?: string
+) => {
+  return OrderModel.addNewOrder({
+    userId: cart.userId,
+    cartId: cart.id,
+    items: cart.items,
```

Comment:
looks like you forgot to create a copy of items: when order is created, it has an empty items array

Reply:
Perhaps I didn't understand correctly, but the list of items is saved in the order. I simply clear the list of products before deleting the card. 
![image](https://github.com/YuliaBarbashova/NodeJsMentoring/assets/45540461/574d4404-6e74-4b42-9e51-bfb8d1f74edb)
![image](https://github.com/YuliaBarbashova/NodeJsMentoring/assets/45540461/df2ca1a3-58e9-4f9b-86f8-11ae097dfcf2)
![image](https://github.com/YuliaBarbashova/NodeJsMentoring/assets/45540461/9d39b04c-d930-463b-b726-ee155b975124)